typo3/Chart.yaml

```diff
@@ -2,4 +2,4 @@
 apiVersion: v1
 name: typo3
 description: Typo3 Configuration
-version: 0.1.2
+version: 0.1.3
```

Comment:
Is that a bugfix you're doing or a feature?

Reply:
It's a feature. Until now you could define only one service, now you can define several with different names.

Reply:
Then you should increase the version to `0.2.0`